File: samples/ruby-server/app/models/Airport.rb
class Airport

	def initialize (airportName, city, country, iata)
		@airportName = airportName
		@city = city
		@country = country
		@iata = iata
	end


	# getter
	def airportName
	  @airportName
	end

	# setter
	def airportName=(value)
	  @airportName= value
	end

	def city
	  @city
	end

	def city=(value)
	  @city = value
	end

	def country
	  @country
	end

	def country=(value)
	  @country = value
	end

	def iata
	  @iata
	end

	def iata=(value)
	  @iata = value
	end

end

File: demos/java-server/src/main/java/com/farandula/models/Airport.java
package com.farandula.models;

import org.springframework.data.annotation.Id;

/**
 * Created by enrique on 1/05/17.
 */
public class Airport implements Cloneable {

    @Id private String _id;

    private String name;
    private String city;
    private String country;
    private String iata;

    public Airport(String name, String city, String country, String iata){
        this.setName(name);
        this.setCity(city);
        this.setCountry(country);
        this.setIata(iata);
    }

    public Object clone(){

        try {

            return super.clone();

        } catch (Exception e){
            e.printStackTrace();
        }

        return null;

    }


    public String getName(){
        return this.name;
    }

    public void setName(String name){
        this.name = name;
    }

    public String getCity(){
        return this.city;
    }

    public void setCity(String city){
        this.city = city;
    }

    public String getCountry(){
        return country;
    }

    public void setCountry(String country){
        this.country = country;
    }

    public String getIata(){
        return this.iata;
    }

    public void setIata(String iata){
        this.iata = iata;
    }


}